ProjectSettings/GraphicsSettings.asset

```diff
@@ -29,6 +29,7 @@ GraphicsSettings:
   m_AlwaysIncludedShaders:
   - {fileID: 4800000, guid: 650dd9526735d5b46b79224bc6e94025, type: 3}
   - {fileID: -6465566751694194690, guid: 9b4e681081e2b4c469111bb649e2f7ee, type: 3}
+  - {fileID: -6465566751694194690, guid: 6a56f4d49ccc01c4caeafc424f9766ed, type: 3}
```

Comment:
What's this? The skybox? If wanted, then fine. :-)

Reply:
This is vinces texture scrolling shader, that is used for the running water.